test/test_docling_doc.py

```diff
@@ -767,6 +780,54 @@ def _construct_doc() -> DoclingDocument:
 
     doc.add_form(graph=graph)
 
+    inline_fmt = doc.add_group(label=GroupLabel.INLINE)
+    doc.add_text(
+        label=DocItemLabel.PARAGRAPH, text="Some formatting chops:", parent=inline_fmt
```

Comment:
Dont use `PARAGRAPH`, we should only use `TEXT`. `PARAGRAPH` comes from a legacy, which we should deprecate (or at least mark as deprecated)